store/postgres/src/notification_listener.rs

```diff
@@ -218,8 +218,10 @@ pub struct JsonNotification {
     pub payload: serde_json::Value,
 }
 
-// Any payload bigger than this is considered large.
-static LARGE_NOTIFICATION_THRESHOLD: usize = 7800;
+// Any payload bigger than this is considered large. Any notification larger
+// than this will be put into the `large_notifications` table, and only
+// its id in the table will be sent via `notify`
+static LARGE_NOTIFICATION_THRESHOLD: usize = 128;
```

Comment:
Is this worth making an env var with the default defined in the code?

Reply:
I don't think that's useful here - I have no idea how to tell people what a good value for it would be. If this helps with long commit times, we'll leave it at this, and if not, I might revert it. But either way, it's not really something that can be tuned.